randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2018,6 +2013,8 @@ Extra - asset_id: collision_camera_025
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Door to Save Station Southeast
       Trivial
+  > Tile Group (MISSILE)
+      Trivial
```

Comment:
Personally, i feel this node can just be deleted. doesnt serve any purpose logically

Reply:
Check